dynalab/tasks/README.md

```diff
@@ -23,31 +23,31 @@ For currently existing tasks, the output format is expected to be
 **nli**
 ```
 {
-    "id": copy from input["uid"],
+    "id": copy from input["uuid"],
```

Comment:
Had a chat with Tristan offline, main rationale behind changing `uid` to `uuid` was to clearly disambiguate `uid` from the `uid` variable that we use elsewhere to mean `user id`. We agreed that this would require additional changes such as translation endpoints for backward compatibility and decided against this change.